pytket/extensions/braket/backends/braket.py

```diff
@@ -391,7 +416,10 @@ def __init__(
         )
 
         paradigm = props["paradigm"]
-        n_qubits = paradigm["qubitCount"]
+        n_qubits = len(self._all_qubits)
+
+        # TODO Is there a programmatic way to discover this (except trial and error)?
+        self._supports_client_qubit_mapping = provider == "rigetti"
```

Comment:
`device_info["disabledQubitRewiringSupported"]` might be it. Here are the values for all available gate based devices.

Lucy: False
ionQdevice: False
Aspen-11: True
Aspen-M-2: True
dm1: None
tn1: None
sv1: None
braket_sv: None
braket_dm: None